game/tests/test_services.py

```
@@ -335,3 +335,84 @@ def test_delete_active_game(self):
         request2 = self.factory.get("/")
         middleware.process_request(request2)
         delete_active_game(request2)  # Should not raise exception
+
+
+# ─── Tests for Issue #2186: game_id session scoping ──────────────────────────
+
+def test_new_game_returns_game_id(self):
+    """POST /api/new-game/ must return a unique game_id field."""
+    response = self.client.post(
+        '/api/new-game/',
+        data=json.dumps({'mode': 'ai'}),
```

Ruff (0.15.20)

[error] Undefined name `json`

(F821)